src/components/ui/TranslationLabel/TLabel.tsx

```diff
@@ -26,12 +26,18 @@ const dictionnariesMap = {
   language: translationLanguageDictionnaries,
 };
 
-type GlobalTranslationLabelType =
+export type GlobalTranslationLabelType =
   | TranslationLabelType
   | TranslationFeatureLabelType
   | TranslationLanguageLabelType
   | TranslationCategoryLabelType;
```

Comment:
```suggestion
type GlobalTranslationLabelType =
  | TranslationLabelType
  | TranslationFeatureLabelType
  | TranslationLanguageLabelType
  | TranslationCategoryLabelType;
```
**Note** : ils sont déjà en export dans le dossier `constants` et le label dans le fichier `translations`